# Request
---
# Response
bool setforce_clb_accepted
